src/main/java/Logout.java

```diff
@@ -0,0 +1,27 @@
+import java.io.*;
+
+import javax.servlet.http.HttpSession;
+import javax.servlet.http.*;
+import javax.servlet.*;
+
+import java.util.HashMap;
```

Comment:
Do we really need to import `java.util.HashMap` in Logout.java servlet? (Line 7)